compareECLFiles.cmake

```diff
@@ -226,18 +228,17 @@ if(MPI_FOUND)
   set(abs_tol_parallel 0.02)
   set(rel_tol_parallel 1e-5)
 
-  foreach(sim flow flow_mpi flow_ebos)
```

Comment:
This removes flow_mpi runs of these two cases, was that also your intention? We intend to remove that program shortly, so it's ok with me.

Total `make test` time for opm-simulators went like this for me:
```
(before)
Total Test time (real) = 215.78 sec
(after)
Total Test time (real) = 121.84 sec
```
Not running these two parallel tests with legacy flow_mpi accounts for most of that reduction.

Reply:
yes, that's what you said on hangout so i did intend to remove the flow_mpi variant.

Reply:
if you think it's premature i can add it back.

Reply:
No it's ok with me!

Reply:
there is also little point in testing `flow_ebos`in addition to `flow` nowadays. (unless you want to test that the single-purpose binaries work.)

Reply:
oops, false alarm. I looked at the removed line, not the added one. sorry for the noise.